lib/beaker/hypervisor/google_compute.rb

```diff
@@ -186,7 +186,7 @@ def provision
         ]
         
         # Check for google's default windows images and turn on ssh if found
-        if image_project == "windows-cloud"
+        if image_project == "windows-cloud" || image_project == "windows-sql-cloud"
```

Comment:
Would it make sense to do this?
```ruby
if image_project.start_with?('windows-')
```

Reply:
If the specific images are important, maybe:

```suggestion
        if ['windows-cloud', 'windows-sql-cloud'].include?(image_project)
```

Reply:
Based on OOB discussion, leaving it with the specific projects and entering a ticket to enable turning this on for other projects/in other situations.

See #28